### PR TITLE
Main page about section

### LINKED ISSUE #54: [Site] Add an About section
**Desired State of the Service Mesh Patterns site**

An `About` needs to be added to the site as per this [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=8621%3A13932) design.

---
### Contributor Resources
This website uses React and GitHub Pages. Site content is found under the [`master` branch](https://github.com/service-mesh-books/service-mesh-patterns/tree/master/src).
- See the [Contributing to Service Mesh Patterns Website](https://github.com/service-mesh-patterns/service-mesh-patterns/#readme) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- Layer5 [Brand Kit](https://layer5.io/company/brand)
- 📑 [Service Mesh Patterns in Meshery Catalog](https://meshery.io/catalog)
- 📖 [Service Mesh Patterns Book](https://layer5.io/learn/service-mesh-books/service-mesh-patterns)
- 🎨 [Site designs in Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=5310%3A0). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by legendarykamal**
**Description**

This PR fixes #54

**Notes for Reviewers**
About sec created as per figma design
image

->Addition of vector at bottom is also done.

image

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: site/src/components/Mainabout.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import '../sections/mainabout.css';
```
Comment (Chadha93) [inline site/src/components/Mainabout.js:2]: @legendarykamal Please use Gatsby styled components instead of plane CSS.

**Comment by legendarykamal** [comment]
Yes @Chadha93 i had tried using it only at first place but faced a bit errors 😅 so shifted to plain ones. 
But I will consider converting this 👍 to styled ones

File: site/src/components/Mainabout.js (hunk 1 above)
Comment (legendarykamal) [reply site/src/components/Mainabout.js:2]: Yes @Chadha93 i had tried using it only at first place but faced a bit errors 😅 so shifted to plain ones. 
But I will consider converting this 👍 to styled ones

**Comment by l5io** [comment]
🚀 Preview for commit a78ecea4104803ccdaceb9a5cdc597fd699d1f4e at: https://62f107b88e5c0c7df5e19a52--service-mesh-patterns.netlify.app

File: site/src/components/Mainabout.js (hunk 1 above)
Comment (legendarykamal) [reply site/src/components/Mainabout.js:2]: @Chadha93 done changing to gatsby styled components

**Comment by legendarykamal** [comment]
Updated SS
image

**Comment by l5io** [comment]
🚀 Preview for commit 24e9a2bba01722eb507db15b2edaed5df5cd1ee6 at: https://62f7c4ac94aca53a57cf5eff--service-mesh-patterns.netlify.app

File: site/src/components/Mainabout.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from 'react'
```
Comment (Nikhil-Ladha) [inline site/src/components/Mainabout.js:1]: The file should be under `sections`.
Create a folder named `About` and add then create an `index.js` file under it and the `about.style.js` file for CSS

File: site/src/components/Mainabout.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import { AboutStyleWrapper } from '../sections/mainabout.style'
+
+export const Mainabout = () => {
+  return (
+    <AboutStyleWrapper>
+        <div className='about'>
```
Comment (Nikhil-Ladha) [inline site/src/components/Mainabout.js:7]: Is this `div` really needed?

File: site/src/components/Mainabout.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            <div className="heading">About the <span className='green'>Service Mesh Patterns</span> Book</div>
```
Comment (Nikhil-Ladha) [inline site/src/components/Mainabout.js:8]: It's heading so use `h1` for it!
And, within it use the `span` for applying CSS to a particular text

File: site/src/components/Mainabout.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+            </div>
+            <br />
+            <div className='Box Box2'>
+                As your organization adopts microservices, you also need a service mesh, the infrastructure layer that handles interservice communication in microservice architectures. This seminal book provides patterns and operational best practices to help you deploy and use service meshes in a way that meets your company's goals and needs.
+            </div>
+            <br />
+            <div className='Box Box3'>
+                Developers face a wide range of choices today: which service mesh to use, which features to deploy, and the best ways to exploit and deploy those features. Whether you're investigating service meshes or already use one, authors Lee Calcote and Nic Jackson provide universal patterns applicable to any service mesh: Istio, Linkerd, Consul, App Mesh, and more.
+            </div>
```
Comment (Nikhil-Ladha) [inline site/src/components/Mainabout.js:19]: Why 3 div's?
These are description texts so use the `p` tag for them.
Create a parent div and inside it use 3 `p` for each paragraph.
Ex:
```
<div>
    <p> ......</p>
    <p> .......</p>
    <p>.........</>
</div>
```

File: site/src/components/Mainabout.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+                <li className='list'>Understand how service meshes benefit developers, operators, and service owners differently.</li>
+                <li className='list'>Determine whether you're running your mesh effectively.</li>
+                <li className='list'>Avoid service mesh lock-in through use of service mesh abstractions.</li>
+                <li className='list'>Learn common deployment patterns for implementing a specific service mesh function.</li>
+                <li className='list'>Use configuration best practices to avoid common pitfalls.</li>
+                <li className='list'>Explore federation of service mesh deployments and other service mesh types.</li>            
+            </ul>
+            <div className="Box">
+                As microservices proliferate your environment, fear not: service meshes are here to manage your service sprawl. This book shows you how.
+            </div>
```
Comment (Nikhil-Ladha) [inline site/src/components/Mainabout.js:32]: Use `p`

File: site/src/components/Mainabout.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+            <div className='subhead'>Who is this book for?</div>
```
Comment (Nikhil-Ladha) [inline site/src/components/Mainabout.js:33]: This is heading but at a sublevel so use `h2` for it.

File: site/src/components/Mainabout.js [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+            <div>
+            <div className="bbox">Identifying who this book is for has been one of the hardest challenges in writing it. The power of a service mesh is vast, and therefore, empowers individuals of many different roles.</div>
+            <br />
+            <div className="bbox">In short, this book is for anyone delivering workloads enabled by a service mesh and who wants to ensure their use of a service mesh provides the most possible value to the workloads and teams that rely on it.</div> 
+            </div>
```
Comment (Nikhil-Ladha) [inline site/src/components/Mainabout.js:38]: Same as above use `p` for these paragraphs.
Ex:
```
<div>
     <p>.....</p>
     <p>......</p>
</div>
```

File: site/src/sections/mainabout.style.js [hunk 9]
```diff
@@ -0,0 +1,200 @@
+import styled from "styled-components";
+
+export  const AboutStyleWrapper = styled.div`
```
Comment (Nikhil-Ladha) [inline site/src/sections/mainabout.style.js:3]: Use `section`, i.e, `styled.section`

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Please use semantic HTML.
Avoid unnecessary/repetitive usage of divs. `div` should be the last resort when no other HTML tag fits.

File: site/src/components/Mainabout.js (hunk 2 above)
Comment (legendarykamal) [reply site/src/components/Mainabout.js]: got it :+1:

File: site/src/components/Mainabout.js (hunk 3 above)
Comment (legendarykamal) [reply site/src/components/Mainabout.js]: No i dont think so , removed it

File: site/src/components/Mainabout.js (hunk 4 above)
Comment (legendarykamal) [reply site/src/components/Mainabout.js]: ah yes should hv used h1 , span is already been used :100:

**Comment by l5io** [comment]
🚀 Preview for commit cd00d148fc54a289d1adf01b904aa2bc7d6e5c8c at: https://62fa13dbd35b2b2aaa7f8da8--service-mesh-patterns.netlify.app

File: site/src/components/Mainabout.js (hunk 7 above)
Comment (legendarykamal) [reply site/src/components/Mainabout.js]: :+1:

**Comment by l5io** [comment]
🚀 Preview for commit 1cf619b55f10823ff4b92c860260544ee1b3efbe at: https://631603df48c9b86a4d09c5b8--service-mesh-patterns.netlify.app

File: site/src/sections/About/index.js [hunk 10]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { AboutStyleWrapper } from './about.style'
+
+const Mainabout = () => {
+  return (
+    <AboutStyleWrapper>
+            <h1 className="heading">About the <span className='green'>Service Mesh Patterns</span> Book</h1>
```
Comment (Nikhil-Ladha) [inline site/src/sections/About/index.js:7]: Change the indentation to 2 spaces for the whole component definition.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by legendarykamal** [comment]
Hi @Nikhil-Ladha I have removed the vertical space and given it full width and now it looks fine .

Have a look .
image

File: site/src/sections/About/about.style.js [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+    height: 45px;
+    font-family: 'Qanelas Soft';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 36px;
+    line-height: 45px;
+    flex: none;
+    order: 0;
+    align-self: stretch;
+    flex-grow: 0;
```
[the remark this answers is not in the document]
Comment (legendarykamal) [reply site/src/sections/About/about.style.js]: done

**Comment by l5io** [comment]
🚀 Preview for commit f1d895b81694afa4aff53baa55ffc2752c9cc536 at: https://6316125bd8aeff73bc33ffdf--service-mesh-patterns.netlify.app

File: site/src/sections/About/index.js [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+            <div>
```
[the remark this answers is not in the document]
Comment (legendarykamal) [reply site/src/sections/About/index.js]: used section :+1:  good point btw

File: site/src/sections/About/index.js [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+                </p>
+                <br />
+                <p className='Box Box2'>
+                    As your organization adopts microservices, you also need a service mesh, the infrastructure layer that handles interservice communication in microservice architectures. This seminal book provides patterns and operational best practices to help you deploy and use service meshes in a way that meets your company's goals and needs.
+                </p>
+                <br />
+                <p className='Box Box3'>
+                    Developers face a wide range of choices today: which service mesh to use, which features to deploy, and the best ways to exploit and deploy those features. Whether you're investigating service meshes or already use one, authors Lee Calcote and Nic Jackson provide universal patterns applicable to any service mesh: Istio, Linkerd, Consul, App Mesh, and more.
```
[the remark this answers is not in the document]
Comment (legendarykamal) [reply site/src/sections/About/index.js]: yes removed .

File: site/src/sections/About/index.js (hunk 10 above)
Comment (legendarykamal) [reply site/src/sections/About/index.js]: done

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
The page is not responsive.
The content should be center aligned.
The book image is getting clipped, that should not happen.

**Comment by l5io** [comment]
🚀 Preview for commit e9a59fd60225898c6b48b38c31457912639345cc at: https://631c703a734bf12bd93a90a1--service-mesh-patterns.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4b6c614f6314f64a803443756c9479ab554552e8 at: https://631c73caf19e9b3332e0474b--service-mesh-patterns.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f40c3de923b056a7071d76bbf71c3bb91953c2f7 at: https://631c9f857bc2084fe9cd7111--service-mesh-patterns.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1591bc7abdc8cb0d36037bbf6d2c6c4327a7780f at: https://631e399cd1b60466e59d7472--service-mesh-patterns.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e536be01151c535c6e8cc52616a8b2a2dca96acd at: https://631ea25df5416c2c3dbbdd14--service-mesh-patterns.netlify.app

**Comment by Nikhil-Ladha** [comment]
I have added a `Container` component recently. So, please use that in the `Author` section code to have the `max-width` set properly.

**Comment by legendarykamal** [comment]
> I have added a `Container` component recently. So, please use that in the `Author` section code to have the `max-width` set properly.

were you referring to this @Nikhil-Ladha  https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=8621%3A13917 ?

**Comment by Nikhil-Ladha** [comment]
Yes, wrap the about section within the `Container`.

**Comment by legendarykamal** [comment]
@Nikhil-Ladha added the container but I am not able to see what the merge conflicts are !

**Comment by Nikhil-Ladha** [comment]
Did you fetch the latest master and rebase correctly?

**Comment by l5io** [comment]
🚀 Preview for commit a0b083fa9d4ea3415e421f928f5cc9001797f41f at: https://63274152e2cf574ae14cb64f--service-mesh-patterns.netlify.app

File: site/src/sections/About/index.js [hunk 14]
```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { AboutStyleWrapper } from "./about.style";
+import Openvector from "../../assets/images/about/Vector.png";
+import Closevector from "../../assets/images/about/vector.png";
+
+const Mainabout = () => {
+  return (
```
Comment (Nikhil-Ladha) [inline site/src/sections/About/index.js:7]: Wrap the whole component within the `Container` component here.

Comment (legendarykamal) [reply site/src/sections/About/index.js:8]: added :tiger:

**Comment by l5io** [comment]
🚀 Preview for commit 01b2a4caa3ac90d27c612bcc93dab24e9bdc77fb at: https://632755743cd2e056b96e1412--service-mesh-patterns.netlify.app

**Comment by Nikhil-Ladha** [comment]
Remove the `coming soon` section and the extra padding applied on the section wrapper, it is not needed after using `Container`.
The "Who is this book for" header has fixed width causing responsive issues, please remove that.

**Comment by l5io** [comment]
🚀 Preview for commit c0fa4e383c058e2a9d9a26c95b7453b94a4c663d at: https://63275fef78b23e5765216ab0--service-mesh-patterns.netlify.app

**Review by debo19 — CHANGES_REQUESTED** [review]
Screenshot 2022-09-19 at 1 21 09 PM
Remove this dark background and make sure the book image overlap is as per the design.

**Comment by legendarykamal** [comment]
> Screenshot 2022-09-19 at 1 21 09 PM
> 
> Remove this dark background and make sure the book image overlap is as per the design.

@debo19 these changes are due to the ```container``` component added that @Nikhil-Ladha asked me to. I guess I will have to make the changes in that.

**Comment by Nikhil-Ladha** [comment]
Well, if the `Coming soon` background is removed we won't see that issue.
And, as far as the overlap is concerned updating the `z-index` of the book image should do the trick.

I doubt any changes to the `Container` component is needed

**Comment by legendarykamal** [comment]
@Nikhil-Ladha i have removed the entire ```coming soon``` section I wonder what is causing the dark bg :thinking:

**Comment by legendarykamal** [comment]
@debo19 can you please tell your zoom ratio for me its not overlapping

**Comment by Nikhil-Ladha** [comment]
> @debo19 can you please tell your zoom ratio for me its not overlapping
It's overlapping for me as well.
Though, have you heard of `Developer Console` ? If so, maybe try using it. Or, else now would be a good time to explore it.

And, the same applies for the dark bg

**Comment by legendarykamal** [comment]
haha yep using it but its showing me that the dark bg is caused due to ```graphic``` image used in ```hero``` section.

**Comment by l5io** [comment]
🚀 Preview for commit 511af41450355ebcd9fd05f9a4a223172a723600 at: https://632842abd6c7a6065790baa3--service-mesh-patterns.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ca0f93e8369b0def3bc74c8cb8813c54cbc73ad1 at: https://6328450be16b5105d58c0ed5--service-mesh-patterns.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 385a53f1048dd848c5a4fe7b8e7550e63a3930c0 at: https://63284b63c4bad00e3975d75b--service-mesh-patterns.netlify.app

**Comment by legendarykamal** [comment]
@debo19  now overlaps perfectly as expected

**Comment by l5io** [comment]
🚀 Preview for commit 02ef28d58e5f51ef445fccf7b9a27013b6ebac32 at: https://632aa24191b63b2639b52a08--service-mesh-patterns.netlify.app

**Pull request merged**